src/Badges.sol

```diff
@@ -148,9 +156,21 @@ contract Badges is
       "giveToMany: unauthorized"
     );
 
+    address[] memory invalidRecipients = new address[](_recipients.length);
+    uint256 invalidRecipientsIndex = 0;
```

Comment:
the idea here is that as we come across invalid signatures, we'll pass them to this array and return an array here. 
Then, on the frontend, if we have an `array.length > 0` we can list out the addresses that didn't mint, and let the frontend know.